server.js

```diff
@@ -305,8 +306,17 @@ app.use('/pl/course_instance/:course_instance_id/instructor/grading_job', requir
 app.use('/pl/course_instance/:course_instance_id/instructor/syncs', require('./pages/courseSyncs/courseSyncs'));
 app.use('/pl/course_instance/:course_instance_id/instructor/jobSequence', require('./pages/instructorJobSequence/instructorJobSequence'));
 app.use('/pl/course_instance/:course_instance_id/instructor/loadFromDisk', require('./pages/instructorLoadFromDisk/instructorLoadFromDisk'));
-app.use('/pl/course_instance/:course_instance_id/instructor/course', require('./middlewares/authzCourseInstanceHasCourseView'));
-app.use('/pl/course_instance/:course_instance_id/instructor/course', require('./pages/courseOverview/courseOverview'));
+
+// admin pages
+app.use(/^(\/pl\/course_instance\/[0-9]+\/instructor\/admin)\/?$/, (req, res, _next) => {
+    res.redirect(`${req.params[0]}/access`);
+});
+app.use('/pl/course_instance/:course_instance_id/instructor/admin/access', [
+    require('./pages/instructorAdminAccess/instructorAdminAccess'),
+]);
+app.use('/pl/course_instance/:course_instance_id/instructor/admin/lti', [
+    require('./pages/instructorAdminLti/instructorAdminLti'),
+]);
```

Comment:
We should also mount `./pages/courseOverview/courseOverview` at the route `/pl/course_instance/:course_instance_id/instructor/admin/course` so that it can appear as a real tab, not as a weird fake tab where you click the tab but then go to a totally different page. This will require some changes to `navbarInstructor.ejs` to render the dropdown correctly for this page, and also some changes to `courseOverview.ejs` to include the tabbar when mounted on the `instructor/admin` route. We can detect which route we are on by either setting a special variable in `server.js` just before the `require`, or by looking for whether `locals.course_instance` is defined.